src/address/index.ts

```diff
@@ -633,8 +633,8 @@ export class AddressDelegated extends Address {
     if (!validateNetwork(network)) throw new InvalidNetwork()
     if (parseInt(protocolIndicator) != ProtocolIndicator.DELEGATED) throw new InvalidProtocolIndicator()
 
-    const namespace = address.slice(2, address.indexOf(network, 2))
-    const dataEncoded = address.slice(address.indexOf(network, 2) + 1)
+    const namespace = address.slice(2, address.indexOf('f', 2))
```

Comment:
Didn't it used to be "f" or "t"?

Reply:
We had a bug there. That is not network indicator. It is a delimiter for namespace and subaddress. If you are interested in reading more, check it [here](https://docs.filecoin.io/smart-contracts/filecoin-evm-runtime/address-types/#delegated-addresses). It is tricky though